Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -87,7 +87,7 @@ def http_request(method, url_suffix, data=None, headers=None, num_of_seconds_to_
                                verify=USE_SSL,
                                data=data,
                                headers=headers,
-                               timeout=15
+                               timeout=REQUEST_TIMEOUT
```

Comment:
The way we handle request timeouts in other integration is via server configuration. And not an integration parameter.
I think it is better to add similar code here in order to be aligned with other integrations.


The following screenshot is from CommonServerPython.
<img width="893" alt="image" src="https://user-images.githubusercontent.com/7270217/192422053-dc3570ba-11ba-4666-92da-06a05bdd6eca.png">